.github/workflows/file-syncer.yaml

```diff
@@ -0,0 +1,20 @@
+name: File Syncer
+on:
+  push:
+    branches:
+      - main
```

Comment:
nit: only run the workflow when you modify the `sync` workflows

Reply:
I was thnking of doing the sync actually on a cron so that if someone mistakenly edits local file will be changed back